EMS/Models/Department.cs

```diff
@@ -0,0 +1,7 @@
+namespace EmployeeManagementSystem.Models;
+
+public class Department
+{
+    public int Id { get; set; }
```

Comment:
You can simply have a single model called Reference or Dropdown that has Id and Name fields.